src/controllers/request.js

```diff
@@ -281,7 +284,7 @@ export default class requestsController {
       const request = await Request.isRequestBelongsToManager(requestId, managerId);
       if (!request) throw 'Request not found!';
       const todayDate = getTodayDate();
-      if (todayDate >= request.departureDate) throw 'The request start date is due.';
+      if (process.env.NODE_ENV !== 'test') if (todayDate >= request.departureDate) throw 'The request start date is due.';
```

Comment:
Can you explain what you are trying to do on this line?

Reply:
@kagabof , to test this feature, I had: 
1. create trips with both departure date and return date in the past.
2. approve those trips.

To achieve this, the only thing I could do is to skip the validations while testing, otherwise, I could not create a request with past dates or approve a request that ended.

On line 287, I was skipping that simple validation on the DB field while we are testing because it is checking whether the trip's departure date that the manager is trying to approve is in the coming dates not in the past dates and yet in our tests we want to approve a request with the past date.

@kagabof , let me know if the approach is understandable.